css/react_paragraphs.field_formatter.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["webpack:///./scss/field_formatter.scss"],"names":[],"mappings":"AAAA;EAEI,WAAW;;AAIf;EACE;IACE,oBAAa;IAAb,oBAAa;IAAb,aAAa;IACb,qBAAiB;QAAjB,iBAAiB;IACjB,yBAA8B;QAA9B,sBAA8B;YAA9B,8BAA8B;IAHhC;MAMI,WAAW;IANf;MAYM,iCAAkC;IAZxC;MAYM,kCAAkC;IAZxC;MAYM,uBAAkC;IAZxC;MAYM,kCAAkC;IAZxC;MAYM,kCAAkC;IAZxC;MAYM,uBAAkC;IAZxC;MAYM,kCAAkC;IAZxC;MAYM,kCAAkC;IAZxC;MAYM,uBAAkC;IAZxC;MAYM,kCAAkC;IAZxC;MAYM,kCAAkC,IACnC","file":"../css/../../css/react_paragraphs.field_formatter.css","sourcesContent":[".react-paragraphs-row {\n  [data-react-columns] {\n    width: 100%;\n  }\n}\n\n@media (min-width: 768px) {\n  .react-paragraphs-row {\n    display: flex;\n    flex-wrap: nowrap;\n    justify-content: space-between;\n\n    [data-react-columns] {\n      width: 100%;\n    }\n\n    @for $i from 1 through 11 {\n      $percentage: percentage($i / 12);\n      [data-react-columns=\"#{$i}\"] {\n        width: calc(#{$percentage} - 30px);\n      }\n    }\n  }\n}\n"],"sourceRoot":""}
```

Comment:
Can we ignore this file?

Reply:
I personally prefer to track map files. They aren't loaded to anyone unless they have inspector open. And what I hate is when i open my inspector on my local and suddenly the database log is full of 404 messages about missing .map files.

And production aggregation strips css map declarations anyways

There's some articles supporting my point of view as well.

Reply:
You have successfully confused me on this topic.